packages/typescript/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   name: "typescript",
-  version: "4.3.5",
+  version: "4.4.0-beta250.0",
```

Comment:
If we are already updating the version here, we should also bump the Typescript version in the tool and meteor-babel and go to 4.4.3 to keep the versioning schema that was started.